other/ERRORS_AND_WARNINGS.md

```diff
@@ -18,3 +18,11 @@ point to a file that actually exists.
 
 This happens if you specify a script name that can't be resolved to a script. See the docs/examples for how you can
 define scripts and how they can be referenced.
+
+## Unable to preload module
+
+This happens the module which you try to preload couldn't be resolved.
```

Comment:
This could probably more clearly call out the relationship with the `--require` flag. Something like:

> This happens when you use the `--require` flag and the module you specify cannot be resolved.